docs/docs/workflows/config.md

```diff
@@ -0,0 +1,228 @@
+  - `Compute Security Admin (roles/compute.securityAdmin)`
+  - `Compute Storage Admin (roles/compute.storageAdmin)`
+  - `Service Account User (roles/iam.serviceAccountUser)`
+
+  Afterward, create and download a new JSON key for this service account. Place the downloaded file in your Constellation workspace, and set the config parameter to the filename, e.g., `constellation-129857-15343dba46cb.json`.
+
+</tabItem>
+
+<tabItem value="aws" label="AWS (manually)">
+
+* **region**: The name of your chosen AWS data center region, e.g., `us-east-2`.
+
+  Constellation OS images are currently replicated to the following regions:
+  * `eu-central-1`
+  * `us-east-2`
+  * `ap-south-1`
+
+  If you require the OS image to be available in another region, [let us know](https://github.com/edgelesssys/constellation/issues/new?assignees=&labels=&template=feature_request.md&title=Support+new+AWS+image+region:+xx-xxxx-x).
+
+  You can find a list of all [regions in AWS's documentation](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-regions-availability-zones.html#concepts-available-regions).
+
+* **zone**: The name of your chosen AWS data center availability zone, e.g., `us-east-2a`.
+
+  Learn more about [availability zones in AWS's documentation](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-regions-availability-zones.html#concepts-availability-zones).
+
+* **iamProfileControlPlane**: The name of an IAM instance profile attached to all control-plane nodes.
+
+  Use the [provided Terraform script](https://github.com/edgelesssys/constellation/tree/release/v2.2/hack/terraform/aws/iam) to generate the necessary profile. The profile name will be provided as Terraform output value: `control_plane_instance_profile`.
+
+  Alternatively, you can create the AWS profile with a tool of your choice. Use the JSON policy in [main.tf](https://github.com/edgelesssys/constellation/tree/release/v2.2/hack/terraform/aws/iam/main.tf) in the resource `aws_iam_policy.control_plane_policy`.
+
+* **iamProfileWorkerNodes**: The name of an IAM instance profile attached to all worker nodes.
+
+  Use the [provided Terraform script](https://github.com/edgelesssys/constellation/tree/release/v2.2/hack/terraform/aws/iam) to generate the necessary profile. The profile name will be provided as Terraform output value: `worker_nodes_instance_profile`.
+
+  Alternatively, you can create the AWS profile with a tool of your choice. Use the JSON policy in [main.tf](https://github.com/edgelesssys/constellation/tree/release/v2.2/hack/terraform/aws/iam/main.tf) in the resource `aws_iam_policy.worker_node_policy`.
+
+</tabItem>
+
+</tabs>
+</details>
+
+Now that you've configured your CSP, you can [create your cluster](./create.md).
+
+## Deleting an IAM configuration
+
+Created IAM configurations can be kept and reused for new clusters. Alternatively, you can also delete them if they aren't being used anymore.
+
+### Prerequisites
```

Comment:
```suggestion
**Prerequisites:**
```
otherwise it creates an entry in the sidebar on the right and then we would need another heding for the actual commands